sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -347,6 +347,10 @@ querySpecification
        windows?)
     ;
 
+intoClause
+    : INTO tableIdentifier
```

Comment:
Could you also check what kind of a plan the following query produces:
```SQL
SELECT a INTO tbl_a FROM tbl_b
```
We might run into a weird syntax error here. If we do then we need to move the `INTO` keyword from the `nonReserved` rule to the `identifier` rule.